### PR TITLE
IAC-713 Doesn't Ask for All Required Information about importing vRА

### PR DESCRIPTION
**Pull request by marianmarinov**
### Description

Installer v 2.27.0 Doesn't Ask for All Required Information about importing vRА.
Shows a prompt to ask for additional information for importing vRA - project name and ID and Organization name and ID.

### Checklist

- [x] Lint and unit tests pass locally with my changes
- [x] I have added relevant error handling and logging messages to help troubleshooting
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)
- [x] I have updated CHANGELOG.md with a short summary of the changes introduced
- [x] I have tested against live environment, if applicable
- [x] I have added relevant usage information (As-built)
- [x] Any structure and/or content vRA-NG improvements are synchronized with vra-ng and ts-vra-ng archetypes
- [x] Every new or updated Installer property is documented in docs/archive/doc/markdown/use-bundle-installer.md
- [x] Dependencies in pom.xml are up-to-date
- [x] My changes have been rebased and squashed to the minimal number of relevant commits
- [x] My changes have a descriptive commit message with a short title, including a `Fixed #XXX -` or `Closed #XXX -` prefix to auto-close the issue

### CODE REVIEW

**Comment by Michaelpalacce** [comment]
Please also add an entry to "Release.md"

File: package-installer/src/main/java/com/vmware/pscoe/iac/installer/Installer.java [hunk 1]
```diff
@@ -764,8 +764,8 @@ private static void loadInputInteractiveMode(Input input) {
             userInput(input, Option.VRANG_IMPORT, "Import vRA8 packages?", true);
           
         }
-    
-		if (input.anyTrue(Option.CS_IMPORT)) {
+
+        if (input.anyTrue(Option.VRANG_IMPORT, Option.CS_IMPORT)) {
```
Comment (Michaelpalacce) [inline package-installer/src/main/java/com/vmware/pscoe/iac/installer/Installer.java:768]: Lets take a look at: https://jira.pscoe.vmware.com/browse/IAC-584 first. 

I think we may need to extract readVrangProperties(input); from this if else and leave it as is

**Review by Michaelpalacce — CHANGES_REQUESTED** [review]
Lets make sure we don't re-introduce a previous issue

**Comment by Michaelpalacce** [comment]
Please include what exactly is missing, which options?

**Comment by marianmarinov** [comment]
> 

The prompt for "Project name:" is missing and entire section of vRA questions (Project ID, Organization Name, Organization ID ...)

**Review by Michaelpalacce — APPROVED** [review]

**Review by Michaelpalacce — CHANGES_REQUESTED** [review]
Release.md

**Comment by marianmarinov** [comment]
> Please also add an entry to "Release.md"

Done

File: docs/versions/latest/Release.md [hunk 2]
```diff
@@ -81,6 +81,19 @@ If you build project with polyglot + typescript-project-all the build will succe
 
 None
 
+### Installer v 2.27.0 Doesn't Ask for All Required Information about importing vRА
+Installer script doesn't show a prompt to ask for additional information for importing vRA - project name and ID and Organization name and ID
+
+#### Previous Behavior
```
Comment (Michaelpalacce) [inline docs/versions/latest/Release.md:87]: Previous and new behavior in the form of:

"It wasn't working"

"now it's working"

doesn't give us any good information as to "why it wasn't working" and "how was it fixed"

**Pull request merged**